doc/cprover-manual/unsound_options.md

```diff
@@ -0,0 +1,37 @@
+# Unsound CLI options for various tools
+
+Be advised that the following command line options to `cbmc` and `goto-instrument`
+have been reported to be unsound:
```

Comment:
Can I ask that we start with a bit more context (reflecting the fact that https://github.com/diffblue/cbmc/issues/6480 had some extensive discussion)? I don't think there even is a universally agreed definition of "unsound," so please try to start with a summary of the conclusion of #6480.

Reply:
Added a paragraph with the conclusions from #6480